site/authors/KunalSin9h.adoc

```diff
@@ -1,9 +1,12 @@
-= About the Author
-:page-layout: author
-:page-author_name: Kunal Singh
-:page-github: KunalSin9h
-:page-twitter: KunalSin9h
-:page-linkedin: kunalsin9h
-:page-authoravatar: ../../images/images/avatars/KunalSin9h.jpg
+---
+---
 
-My name is Kunal and I'm currently pursuing a degree in Computer Science and Engineering. I do Full Stack Web Development, using technologies like React, NextJS, TailwindCSS,TypeScript and Go. I'm also passionate about Cloud computing, Backend and Databases.
+---
```

Comment:
Pleaes undo the changes you have introduced to this and other **existing** files